src/swap/v2/index.ts

```diff
@@ -298,6 +313,20 @@ function getFixedOutputSwapQuote({
     decimals
   });
 
+  if (assetOutAmount > outputSupply) {
+    throw new SwapQuoteError(
+      "Output amount exceeds available liquidity.",
+      SwapQuoteErrorType.OutputAmountExceedsAvailableLiquidity
+    );
+  }
+
+  if (swapInputAmount < 0) {
```

Comment:
It returns a negative value when the swapInputAmount is greater than the input supply, we can leave it like that for now but should check it later.